docker/iroh-fedimintd/docker-compose.yaml

```diff
@@ -10,8 +12,22 @@ services:
       - FM_BITCOIN_NETWORK=signet
       - FM_BITCOIN_RPC_KIND=esplora
       - FM_BITCOIN_RPC_URL=https://mutinynet.com/api
-      - FM_BIND_UI=0.0.0.0:8175
+      - FM_BIND_P2P=0.0.0.0:8173
+      - FM_BIND_API_IROH=0.0.0.0:8174
+      - FM_BIND_API_WS=172.25.0.11:8174
```

Comment:
We should still bind the api plaintext to internal docker network, so it doesn't accidentally get publically exposed.